src/MyPlot/forms/subforms/FillForm.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types=1);
+namespace MyPlot\forms\subforms;
+
+use dktapps\pmforms\CustomFormResponse;
+use dktapps\pmforms\element\Input;
+use MyPlot\forms\ComplexMyPlotForm;
+use MyPlot\MyPlot;
+use pocketmine\Player;
+use pocketmine\utils\TextFormat;
+
+class FillForm extends ComplexMyPlotForm {
+    public function __construct(Player $player) {
+        $plugin = MyPlot::getInstance();
+
+        if(!isset($this->plot))
+            $this->plot = $plugin->getPlotByPosition($player);
+
+        parent::__construct(
+            TextFormat::BLACK.$plugin->getLanguage()->translateString("form.header", [$plugin->getLanguage()->get("fill.form")]),
+            [
+                new Input(
```

Comment:
Possibly add a second option as a dropdown menu with block string names and check which is value was entered in the form response?
I can make this change after the PR is merged if necessary

Reply:
I may add this when everything else works.